### PR TITLE
adjusted a few little things

### PR DESCRIPTION
**Pull request by stufima**
Hallo Carl,
wie über WhatsApp angekündigt.
@Robsrepo42: Solltest Du freigeben, informiere bitte Carl.
VG
Marcus

### CODE REVIEW

File: lubridate.Rmd [hunk 1]
```diff
@@ -617,18 +617,32 @@ With indicator and renamed column
...
 ```
 
 Now we apply the knowledge with the indicator to the selection of autumn.
-Now we examine the duration of rainfall in autumn.
+First we examine the duration of rainfall in autumn.
 ```{r sum NIEDERSCHLAGSDAUER - condition - on an interval of season autumn}
-df_sum_autmn <- df_group_autmn %>% 
-  summarise(DAUER = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
-  arrange( STATIONS_ID , JAHR, MONAT, TAG)
+df_sum_autmn_group <- df_group_autmn
```
Comment (carlschmidt26) [inline lubridate.Rmd]: Should it be <code>df_sum_<b>autumn</b>\_group</code> rather than <code>df_sum_<b>autmn</b>_group</code> or are you abbreviating something?

File: lubridate.Rmd [hunk 2]
```diff
@@ -617,18 +617,32 @@ With indicator and renamed column
...
-  summarise(DAUER = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
-  arrange( STATIONS_ID , JAHR, MONAT, TAG)
+df_sum_autmn_group <- df_group_autmn 
+df_sum_autmn <- df_sum_autmn_group %>% 
+              summarise(DAUER = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
+              filter(DAUER>0) %>%
+              arrange( STATIONS_ID , JAHR, MONAT, TAG)
 head(df_sum_autmn, 10) 
 ```
+Now we examine the milliliter of rainfall.
```
Comment (carlschmidt26) [inline lubridate.Rmd:635]: From the climate scientist perspective I would go with _precipitation_ rather than _rainfall_. The former also includes snow, hail, etc. 
In German it's like _Niederschlag_ vs _Regen_. 
But I might sound picky here as this is not the point of your chapter.

**Review by carlschmidt26 — COMMENTED** [review]
Hi Marcus,

thank you for these changes. I haven't started looking into your chapter, so there is no problem with that at all. I added two minor comments in the code. Would be nice if you could give me a feedback on the first one. But I guess I will also find out when I am going through your chapter top down.

One thing I would like to mention, though is:
In climate science one uses the "lazy" approach to define seasons by simply using moths.
**Winter** is often referred to as **DJF** (December, January, February)
**Spring** is often referred to as **MAM** (March, April, May)
...

Basically this ist the difference between e.g. meteorological and astronomical seasons.
Additional information can be found at the [NOAA's NCEI site].(https://www.ncei.noaa.gov/news/meteorological-versus-astronomical-seasons)

This isn't helpful for demonstrating functions of the lubridate package, but it generally makes live easier when working with climate or weather data.

Best,
Carl

**Comment by Robsrepo42** [comment]
all clear and approved
